### ISSUE TITLE
how could i use my dataset to train Sa2VA

### ISSUE THREAD

**Issue by LORDPQK** [bytedance/Sa2VA#46]
i have a png image dataset . Did anyone tried adding their own data for training?
dataset_root/
├── images/
│   ├── 0001.jpg
│   ├── 0002.png
│   └── ...
├── masks/
│   ├── 0001.png  （0=background，1=first object，2=second object...）
│   ├── 0002.png
│   └── ...

**Comment by mrdbourke** [comment]
+1 to this, a fine-tuning guide would be epic!

**Comment by lxtGH** [comment]
Thanks for your attention. We will update this feature in the next two weeks. Please stay tuned.

**Comment by bill-hx-liu** [comment]
+1 to this,thanks

**Comment by LORDPQK** [comment]
@lxtGH Hi,

I wonder if the fine-tuing script of Sa2VA-8B is already released?

thank you!

**Comment by LemonWei111** [comment]
> 感谢您的关注。我们将在接下来的两周内更新此功能。敬请期待。

3 weeks have passed.

**Comment by HarborYuan** [comment]
Hi @LORDPQK @LemonWei111 ,

Thanks for your interest in our work. Your mentioned dataset is like instance segmentation. Sa2VA currently focuses on tasks such as chat and referring segmentation of images and videos. The dataset you mentioned is more like a pure visual task.

For the training guide, I will add it now. Please stay tuned.

**Comment by abhigoku10** [comment]
@HarborYuan  Has the training script and steps updated ???

**Comment by eslambakr** [comment]
Hi @lxtGH 

Thanks for your responses. But it seems releasing finetuning codes will take more time than expected.
Therefore, it would be appreciated if you could give us an exact estimation.

Also, if you think it will not be possible due ot maybe legal or licensing issues, it would be great to know as well.

Thanks again for your efforts!
